styleguide/source/assets/scss/01-atoms/_image.scss

```diff
@@ -35,3 +35,15 @@ figure.embed-entity--caption {
     caption-side: bottom;
   }
 }
+
+figure.embedded-entity.align-left {
+  margin-top: $gutter / 2;
```

Comment:
Margin-top is making the space around the entity uneven due to the fact that paragraph already has a margin bottom of 14px

<img width="917" alt="spacing" src="https://user-images.githubusercontent.com/2271747/60067446-2fc03380-96d0-11e9-9c69-6a53c11c4d67.png">